api/cloud_accounts.go

```diff
@@ -211,3 +218,22 @@ func (svc *CloudAccountsService) update(guid string, data interface{}, response
 	apiPath := fmt.Sprintf(apiV2CloudAccountFromGUID, guid)
 	return svc.client.RequestEncoderDecoder("PATCH", apiPath, data, response)
 }
+
+func (svc *CloudAccountsService) listByType(caType cloudAccountType, response interface{}) error {
```

Comment:
what's the benefit of `listByType` vs. just having this incorporated into `ListByType` ?

Reply:
No worries.  Just didn't seem to be adding much value (i.e. not consumed by anything else in the API package).

Reply:
This is pretty similar to the existing get func, since the api path consts are both "v2/CloudAccounts/%s"

If this comment is implemented, the existing get func could be reused https://github.com/lacework/go-sdk/pull/784/files#r865763917

Reply:
re-worked the get function and removed the `listByType` function. We now use `get` within `ListByType `